### PR TITLE
Widescreen Titlescreens and some minor edits

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Changes our game to support widescreen titles, i'm working with the headmins to add these titles to the config as well.

Also:
title screen walls can no longer be examined
title screen itself can still be examined so admins can change it for the memes

## Why It's Good For The Game

The pictures speak for themselves on this one.

<details>
  <summary>Before</summary>
  
  image  
</details>

<details>
  <summary>After</summary>
  
  image  
</details>

## Changelog

:cl:
add: Titlescreens are now widescreen'd up.
qol: The titlescreen is no longer described as "Effectively impervious to conventional methods of destruction." and only the splash screen can be interacted with
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
FAQ: Will the title still work if i'm not using widescreen?

A: Yes

**Comment by LemonInTheDark** [comment]
does it support both wide and non wide title screens? cause that shouldn't be too hard of a thing to do, and I think would be worthwhile

**Review by Krysonism — APPROVED** [review]
title screen looks good

File: _maps/map_files/generic/CentCom.dmm [hunk 1]
```diff
@@ -251,11 +251,11 @@
 /turf/open/space/basic,
 /area/space)
 "fX" = (
-/turf/closed/indestructible/riveted,
+/turf/closed/indestructible/start_area,
 /area/start)
 "fY" = (
 /obj/effect/landmark/start/new_player,
-/turf/open/floor/plating,
+/turf/closed/indestructible/start_area,
```
Comment (Ghommie) [inline _maps/map_files/generic/CentCom.dmm:258]: mobs can't breath in closed turfs without an air tank. Are you sure this won't be a problem?

Comment (tralezab) [reply _maps/map_files/generic/CentCom.dmm:258]: i... maybe want to go back on this? just in case

Comment (LemonInTheDark) [reply _maps/map_files/generic/CentCom.dmm:258]: decided?

**Review by Maurukas — APPROVED** [review]

File: code/game/turfs/closed/_closed.dm [hunk 2]
```diff
@@ -67,6 +69,11 @@ INITIALIZE_IMMEDIATE(/turf/closed/indestructible/splashscreen)
 	SStitle.splash_turf = src
 	if(SStitle.icon)
 		icon = SStitle.icon
+	var/icon/size_check = icon(icon, icon_state)
+	var/x_to_add = (size_check.Width() - 480) * 0.5
+	if(x_to_add > 0)
+		pixel_x += x_to_add
+	..()
```
Comment (LemonInTheDark) [inline code/game/turfs/closed/_closed.dm:76]: Why are you calling parent twice?

**Pull request merged**